CHANGELOG.rst

```diff
@@ -6,6 +6,11 @@ This document describes changes between each past release.
 1.8.1 (unreleased)
 ------------------
 
+**Bug fixes**
```

Comment:
No I just meant that In the changelog it should not be in the Bug category.

I think it is a feature we want (afaik I want it and @n1k0 want it too)